extension/src/codeLensContainer.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ConfigurationChangeEvent, Disposable, languages, workspace, WorkspaceConfiguration } from 'vscode';
+import { testCodeLensProvider } from './codeLensProvider';
+
+const JAVA_TEST_SETTINGS_CONFIGURATION: string = 'java.test.settings';

```

Comment:
Might be better to put them into https://github.com/Microsoft/vscode-java-test/blob/master/extension/src/constants/configs.ts